build/checks/spelling.py

```diff
@@ -12,11 +12,13 @@
 most = {}
 
 for artist in filter(lambda x: os.path.isdir(os.path.join(ROOT, x)), os.listdir(ROOT)):
+    if artist == "reginabellringers.ca": continue
```

Comment:
Out of scope for this PR, but this should eventually replaced by a flag in the artist.json file. Just putting it here as a note.